### DISCUSSION TITLE
Unable to install ImageMagick-7.1.0-4.x86_64.rpm on RHEL 7.3

### DISCUSSION THREAD

**Discussion by a-sarraf** [ImageMagick/ImageMagick#4267] [Help]
Hello Members,
**I wanted to know which version of ImageMagick is compatible for REHL  7.3.**
 
I am trying  to install ImageMagick-7.1.0-4.x86_64.rpm on following RHEL server 
        **Operating System: Red Hat Enterprise Linux Server 7.3 (Maipo)
       CPE OS Name: cpe:/o:redhat:enterprise_linux:7.3:GA:server
            Kernel: Linux 3.10.0-957.21.3.el7.x86_64
            Architecture: x86-64**
But getting errors like below and many more missing libraries with it's lib package as well.
error: Failed dependencies:
        ImageMagick-libs(x86-64) = 7.1.0-4 is needed by ImageMagick-7.1.0-4.x86_64
        libMagickCore-7.Q16HDRI.so.10()(64bit) is needed by ImageMagick-7.1.0-4.x86_64
        libMagickWand-7.Q16HDRI.so.10()(64bit) is needed by ImageMagick-7.1.0-4.x86_64
        libMagickWand-7.Q16HDRI.so.10(VERS_10.0)(64bit) is needed by ImageMagick-7.1.0-4.x86_64
        libltdl.so.7()(64bit) is needed by ImageMagick-7.1.0-4.x86_64
        libraqm.so.0()(64bit) is needed by ImageMagick-7.1.0-4.x86_64

**Comment by urban-warrior** [comment]
The ImageMagick RPM binaries are built on CentOS 8.4.  If the binaries fail due to dependencies, you can always build from the RPM source or TAR source file.
